resources/run.sh

```diff
@@ -55,15 +54,14 @@ while sleep 30; do
   GRAFANA_STATUS=$?
   ps aux |grep lachesis_exporter |grep -q -v grep
   LACHESIS_EXPORTER_STATUS=$?
-  echo "Node_exporter: $NODE_EXPORTER_STATUS"
-  echo "Prometheus: $PROMETHEUS_STATUS"
-  echo "Grafana: $GRAFANA_STATUS"
-  echo "Lachesis: $LACHESIS_EXPORTER_STATUS"
-  # If the greps above find anything, they exit with 0 status
-  # If they are not both 0, then something is wrong
-  if [ $NODE_EXPORTER_STATUS -ne 0 -o $LACHESIS_EXPORTER_STATUS -ne 0 ] ||
-     [ $GRAFANA_STATUS -ne 0 -o $PROMETHEUS_STATUS -ne 0 ]; then
-    echo "One of the processes has already exited."
-    exit 1
```

Comment:
Was this the reason why it crashed and didn't recover?

Reply:
No, the reason was weird bug in lachesis exporter, which only occurred when running in docker and not when executing binary on host.